src/style.css

```diff
@@ -4,37 +4,54 @@
 }
 
 body {
```

Comment:
- [ ] Greate job implementing the app, however, you should follow the general layout design, kindly reverse your sections so that the leaderboard list is on the left and the form for adding new scores to the right

|Your app design|Wireframe|
|-|-|
|![Screenshot_20220625_174943](https://user-images.githubusercontent.com/25227022/175783142-27153964-83fe-4034-9ff6-27495e334aec.png)|![leaderboard_wireframe](https://user-images.githubusercontent.com/25227022/175783208-c600f3f5-14ef-438e-ab35-58704d2c30c3.png)|